packages/app-builder-lib/src/util/appFileCopier.ts

```diff
@@ -180,7 +181,8 @@ function validateFileSet(fileSet: ResolvedFileSet): ResolvedFileSet {
 
 /** @internal */
 export async function computeNodeModuleFileSets(platformPackager: PlatformPackager<any>, mainMatcher: FileMatcher): Promise<Array<ResolvedFileSet>> {
-  const deps = (await platformPackager.info.getNodeDependencyInfo(platformPackager.platform).value) as Array<NodeModuleInfo>
+  const deps = (await getNodeModules(platformPackager.info.appDir)) as NodeModuleInfo[]
```

Comment:
Do we need the `as NodeModuleInfo[]`? Seems like it's already typed correctly for the function as `Promise<NodeModuleInfo[]>`

Note, I always try to avoid `as SomeType`. Instead, use `const deps: NodeModuleInfo[]` to make sure the compiler flags any errors on unexpected return types